packages/checkout/sdk/src/gasEstimate/bridgeGasEstimate.ts

```diff
@@ -21,5 +22,7 @@ export async function getBridgeFeeEstimate(
     gasMultiplier: 1.1,
     sourceChainId: fromChainId.toString(),
     destinationChainId: toChainId.toString(),
+    token: 'NATIVE',
```

Comment:
We have a constant `NATIVE` that can be used here instead of the actual string. Note this constant is lowercased, so if it has to be uppercased we can use `NATIVE.toUpperCase()`

Reply:
PR updated